metrics/storage/fio_job.sh

```diff
@@ -131,9 +146,9 @@ function parse_fio_results()
 		die "bandwidth result not found: check FIO job configuration"
 	fi
 
-	units=$(echo "$bw_data" grep "bw (" | grep -o "[A-Z]*")
-	avg=$(echo "$bw_data" | awk -F "," '{print $4}' | cut -d "=" -f2)
-	stdev=$(echo "$bw_data" | awk -F "," '{print $5}' | cut -d "=" -f2)
+	units=$(echo "$bw_data" | sed 's/^.*bw (/bw (/' | cut -f1 -d ':' | cut -f2 -d '(' | cut -f1 -d ')' | tail -1)
```

Comment:
Just out of interest, why the change in the parsing? Ah, maybe this is because we added the periodic status update output? Curious, as it adds a bunch of extra command calls in, that's all.
Hmm, I wonder where the periodic status output comes out - if it is stdout and we are capturing it here, then maybe that would/will not fix the 'jenkins timeout due to no activity' (as we will consume it, and Jenkins will not see it...) - let's keep an eye on that...

Reply:
@grahamwhaley yes as we put the periodic status update output and it the test is taking the stdout of every single result, I needed to change the parsing because we were having multiple results displayed and saved in the .csv

Reply:
@grahamwhaley even if the result is displayed in the stdout the jenkins metrics is still failing...should I take another approach?

Reply:
@GabyCT I suspect the problem right now is that the output is *not* displayed on stdout when running - if you check the code where you added the status output, it ends up in `FIO_RUN_JOB`, which is invoked such as:

```
output=$(docker exec ${CONTAINER_ID} bash -c "$FIO_RUN_JOB")
```

So, the output gets captured, and not output. If you look in the logs you can see:
```
07:44:58 
07:44:58 Run status group 0 (all jobs):
07:44:58    READ: io=218976KB, aggrb=218757KB/s, minb=218757KB/s, maxb=218757KB/s, mint=1001msec, maxt=1001msec
07:44:58 
07:49:58 Build timed out (after 5 minutes). Marking the build as aborted.
07:49:58 Build was aborted
```

You can see the 5 minute gap there with no output before the timeout. The reason we do see output from the 'prep' job above there in the log is because that is executed as:
```
docker exec ${CONTAINER_ID} bash -c "$FIO_PREP_JOB"
```
aka - output not captured in the script.

Unless we can dream up some fancy way to both capture and output the log (hmm, maybe something with `tee` and stderr???), then I don't really see an alternative but for me to bump the 'no output timeout'. Remind me - is the run time per-test currently 300+60s (to take into account ramp time as well)? So, if I bump to 10minutes, we should be OK??